using_images/other_images/jenkins.adoc

```diff
@@ -52,18 +52,96 @@ You can find https://github.com/openshift/origin/tree/master/examples/image-stre
-The default login is `*admin/password*`.
+The default user login is `*admin/password*`.
 The default password can be configured by setting the `*JENKINS_PASSWORD*` environment variable.
 
 The following command creates a new Jenkins xref:../../architecture/core_concepts/pods_and_services.adoc#pods[pod] with Jenkins running in a container:
 
+If you choose to run with standard Jenkins authentication while the {product-title} Jenkins image runs inside of {product-title}, the the password for
+`*admin*` will be a randomly generated string.  If you manually instantiate a template to launch Jenkins, the password will be noted in the output.
+Otherwise, you can retrieve it by looking at the environment variable values defined on the Jenkins deployment configuration in the web console or by running:
+
+        $ oc env dc/jenkins --list | grep JENKINS_PASSWORD
+
 ----
 $ oc new-app -e \
     JENKINS_PASSWORD=<password> \
     openshift/jenkins-1-centos7
 ----
 
+==== {product-title} OAuth authentication
+
+{product-title} OAuth authentication will be used when it is explicitly configured in the `*Configure Global Security*` panel in the Jenkins UI, or it will be used automatically when the OpenShift Login plugin
+determines that it is running in a pod and the `*OPENSHIFT_ENABLE_OAUTH*` environment variable on the Jenkins `*Deployment Config*` is set to anything other than `false`.  The plugin will retrieve the
+configuration information needed either from data seeded into the pod, or by interacting with the {product-title} API server via REST invocations.  There are configuration fields available for the plugin, but
+they should only be required for development testing or atypical configurations accessing a different {product-title} installation for OAuth than the installation hosting Jenkins.
+
+The login process is comprised of a series of HTTP redirects.  Rather than the traditional Jenkins login page, you will first be directed to an {product-title} login page, similar to what you see when you log into the web console.  If login
+is successful, you will be redirected to the page within the Jenkins UI you are trying to access.
+
+Valid user / password combinations are controlled by the identity provider in use with {product-title}.  So, for example,  if the default identity provider `Allow All` is used, you can provide any non-empty string for both the user name
+and password.  If you supply the user name used to create the `*Project*` Jenkins is running in, you will have administration privileges.
+
+If one of the other identity providers is running, valid user/password combinations stored with your identity provider must be used to login.
+
+Valid users are automatically added to the Jenkins authorization matrix as part of login.
+
+The {product-title} `*Role*` of the user then dictates which permissions within the Jenkins authorization matrix are associated with the user.  The `admin` `*Role*` maps to the set of
```

Comment:
This gets into the scenario Clayton wasn't interested in documenting when I asked him (you were either traveling from Brno or on PTO when I pinged him)- that it just needed to be possible for a developer to hack it.  You have to explicitly configure the token, cert, namespace, client id, redirect url, and api master url.  And in the case of the namespace and the cert, I have not improved upon what Clayton put together / pulled from the google version of this, and those items have to be mounted in from a file system when you `docker run ...` the image outside of OpenShift.

Do you want to resurface this discussion / put more spit and polish on the outside OpenShift use case.  Or minimally put some disclaimer or something here?